clients/js/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "wormhole-client",
   "version": "1.0.0",
   "dependencies": {
-    "@certusone/wormhole-sdk": "^0.3.0",
```

Comment:
how does this work if the sdk is not a dependency? 🤔

Reply:
Dang it. I didn't mean to delete that line. I meant to delete the line that was pointing at the local version. Putting this back.